File: server/Models/Ingredient.cs
namespace checkpoint_wk10.Models;

public class Ingredient 
{
  public int Id { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime UpdatedAt { get; set; }
  public string Name { get; set; }
  public string Quantity { get; set; }
  public int RecipeId { get; set; }
  public Recipe Recipe { get; set; }
}

File: client/src/models/Ingredient.js
export class Ingredient {
  constructor(data) {
    this.id = data.id
    this.quantity = data.quantity 
    this.name = data.name
    this.recipeId = data.recipeId
    this.createdAt = new Date(data.createdAt)
    this.updatedAt = new Date(data.updatedAt)
    
  }
}